arm/Microsoft.Web/sites/deploy.bicep

```diff
@@ -20,30 +20,16 @@ param clientAffinityEnabled bool = true
 @description('Optional. Configuration of the app.')
 param siteConfig object = {}
 
-@description('Optional. Required if functionapp kind. The resource ID of the storage account to manage triggers and logging function executions.')
+@description('Conditional. Required if app of kind functionapp. Resource ID of the storage account to manage triggers and logging function executions. If provided, the `AzureWebJobsStorage` setting is added to the app\'s app settings.')
```

Comment:
Also here, no reference of the storageAccountId in parameter files of kind functionapp

Reply:
Updated